pkg/create/templates/docker.go

```diff
@@ -3,38 +3,56 @@ package templates
 // Dockerfile is a default Dockerfile laid down by s2i create
 const Dockerfile = `
 # {{.ImageName}}
-FROM openshift/base-centos7
-
-# TODO: Put the maintainer name in the image metadata
-# MAINTAINER Your Name <your@email.com>
-
-# TODO: Rename the builder environment variable to inform users about application you provide them
-# ENV BUILDER_VERSION 1.0
-
-# TODO: Set labels used in OpenShift to describe the builder image
-#LABEL io.k8s.description="Platform for building xyz" \
-#      io.k8s.display-name="builder x.y.z" \
-#      io.openshift.expose-services="8080:http" \
-#      io.openshift.tags="builder,x.y.z,etc."
-
-# TODO: Install required packages here:
-# RUN yum install -y ... && yum clean all -y
+# Here you can use whatever base image is relevant for your application.
+FROM centos:centos7
+
+# Here you can specify the maintainer for the image that you're building
+MAINTAINER Code Guru <codeguru@example.com>
```

Comment:
Who? I'd prefer something closer to `MAINTAINER Your Name <your.name@example.com>`.